### ISSUE TITLE
Fat menu A11y issues

### ISSUE THREAD

**Issue by ichim-david** [kitconcept/volto-light-theme#264]
- [x] There is no indication that the buttons are collapsed which tells you that if you access it they will be expanded and you will get the fat menu
- [x] Close button isn't accessible by tab keyboard
- [x]  Menu isn't closed by escape or tabbing away from menu (only by clicking outside the menu area)

Mobile issues:
- [x] The close and open button have no indication that the button is expanded or collapsed
- [x]  you cannot navigate with the keyboard to the mobile menu since it's just list items
![mobile-navigation](https://github.com/kitconcept/volto-light-theme/assets/152852/42f2c86c-7460-4f2a-a7aa-bcf5f577cc68)

**Comment by tisto** [comment]
@ichim-david thanks for the bugreport! Could you image creating a PR and helping with solving the problem?

@sneridagh do we run a11y tests against Volto Light Theme? We should find a11y violations as early in the process as possible.

@ThomasKindermann and @Molochem are currently working on an a11y report for one of our clients that use Volto Light Theme. If we introduce new violations via add-ons, this causes problems.

**Comment by ichim-david** [comment]
> @ichim-david thanks for the bugreport! Could you image creating a PR and helping with solving the problem?
> 
> @sneridagh do we run a11y tests against Volto Light Theme? We should find a11y violations as early in the process as possible.
> 
> @ThomasKindermann and @Molochem are currently working on an a11y report for one of our clients that use Volto Light Theme. If we introduce new violations via add-ons, this causes problems.

For starters my contribution will be with issue reporting but along the way I hope to be able to also fix some of them with pull requests depending on the workload and the priorities we have when trying the light theme.

@tisto  I think you should have some contributor role going for this repo and give to ppl that want to contribute, for instance I can't even assign this issue to myself if I were to actually work on it.

**Comment by tisto** [comment]
@ichim-david right, this is indeed a problem. You can always fork and create a PR. However, anyone should be able to create tickets. Ideally, we would grant Plone contributors and collective users further permissions.

Could not find the proper setting yet. @ericof @sneridagh @davisagli @fredvd ideas?

**Comment by davisagli** [comment]
@tisto I don't think there is a way to grant permissions on the repository to a team from a different github organization.

We can create a Plone Community team inside the kitconcept organization and add people to it as needed, or share specific repositories with specific people. But, I don't currently have permission to manage the permissions for this repository.

**Comment by fredvd** [comment]
2 issues here I think:
* Unless you can affort Github Enterprise, there is no fine grained control over permissions.  As a free or team sponsored account (plone org). you can only assign 5 roles to users. 

* There seems to be a possibility to add 'outside collaborators' to a repository, where those accounts are not a member of the organisation. I just experimented/tested  to try to add a collective team to a plone org repository, but it cannot find the team in the search modal. Or add an individual 'non member' user. So this is maybe also restricted to paying plans/enterprise?

**Comment by davisagli** [comment]
@fredvd I'm not sure what you are trying to do. I was trying to give @ichim-david access to kitconcept/volto-light-theme, not to something in the plone organization. I already did this by adding him as a collaborator on kitconcept/volto-light-theme. (It can't be done with a team, because adding someone to a team requires making them a member of the organization.)

But that is also less important now, because I also fixed the CI so that it will run when we get a PR from a fork.

**Comment by sneridagh** [comment]
@iRohitSingh please take care of this, ask @ichim-david if needed.

**Comment by iRohitSingh** [comment]
Doing.
